chen_formula.py

```diff
@@ -0,0 +1,91 @@
+from math import ceil
+from pypokerengine.engine.card import Card
+
+# Do not use `Card.RANK_MAP.get(12)` as it would return the Card's character, e.g. 1,2,...10,J,Q,K,A
+ACE = 14
+KING = 13
+QUEEN = 12
+JACK = 12
```

Comment:
Typo on value of JACK. 
JACK = 11

Reply:
Thanks for spotting it, will fix. Will also add additional test case for it.